README.md

```diff
@@ -5,12 +5,31 @@
 
 harvest-client is a Java client for the Harvest REST API (https://help.getharvest.com/api-v2/)
 
+## Getting started with Development
```

Comment:
I prefer to have this information in the wiki, I will move it there

Reply:
I like to have README's that get me up and running without going to the Wiki. The Wiki is ok for further information IMO.
If you look at "README best practices", they all have a very similar structure and one thing they have in common is, that they show you how to get started with development.

But there is no reason not have that in the WIKI as well with more detailed information.

Reply:
One example of a README structure that you find very often on OSS projects:

https://gist.github.com/PurpleBooth/109311bb0361f32d87a2

Reply:
I think a README should have all the information for a user of the library to start using it,
but a potential contributor to harvest-client can find more information in the wiki.
Usually there would be many more users than contributors, and they would not care about how to develop the library

For example I like the structure of https://square.github.io/retrofit/